config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :users
```

Comment:
`resources :users` will add by default all standard routes `[show, index, new, create, update, edit, delete]` in cases when you only have a couple of them in your controller you restrict it to only those that are used
http://guides.rubyonrails.org/v5.1/routing.html#restricting-the-routes-created